src/analyze/analyze-security.c

```diff
@@ -1351,7 +1351,7 @@ static const struct security_assessor security_assessor_table[] = {
                 .parameter = CLONE_NEWUSER,
         },
         {
-                .id = "RestrictNamespaces=~CLONE_NEWNS",
+                .id = "RestrictNamespaces=~mnt",
                 .json_field = "RestrictNamespaces_CLONE_NEWNS",
```

Comment:
Shouldn't this be changed to match?

Reply:
I wondered that while making this change. My thought was that `id` is actually shown in the output and seems to imply what the "correct" configuration value is. `json_field` is actually treated more like a documented unique identifier. It also seems to be forming an API of sorts so I didn't want to change it arbitrarily. Looking at #20525, `tests/unit/testsuite-63.sh` and `man/systemd-analyze.xml` would need to be updated if `json_field` was changed. I didn't see any implications of stability around the `id` field which is why I modified only that for these items.

Happy to also change `json_field` if it should be changed; I just wanted to provide context for why I didn't.

Reply:
it's new so it can be changed, it wasn't released yet

Reply:
Followed a pattern similar to `SystemCallFilter` of dropping `~` and other special characters.